### DISCUSSION TITLE
How can I change the embedding & reranking model ?

### DISCUSSION THREAD

**Discussion by weber0904** [open-webui/open-webui#6393] [Q&A]
1. 
I want to use cde-small-v1 ([https://huggingface.co/jxm/cde-small-v1]). It's also in SentenceTransformer package.
![image](https://github.com/user-attachments/assets/b61ccfe9-d900-43d9-8268-5ef991a961c0)

but there show some error I didn't understand.

![image](https://github.com/user-attachments/assets/f811093c-b191-47f2-a697-5cb6c7dcc519)

2. 
Where can I find the Reranking Model list which can show me what model I can use? I find discuss said open-weui can only use the default Reranking Model in SentenceTransformer.

Thanks everyone !

**Comment by flefevre** [comment]
good question...
and woulod it be possible to add a LITELLM connector to manage the Embedding Model and Reranking Model ?
the user interface should allow to enter an url to an openai compatible server .
Thanks for your help
